src/main/java/de/komoot/photon/query/PhotonQueryBuilder.java

```diff
@@ -50,29 +50,32 @@
     private MatchQueryBuilder defaultMatchQueryBuilder;
 
     private MatchQueryBuilder languageMatchQueryBuilder;
+    
+    private MatchQueryBuilder numricMatchQueryBuilder;
 
```

Comment:
This variable isn't used anywhere , and should be removed or used (PS it would "numeric").

Reply:
Thank you very much! I removed the unused the parameter. By the way, could you please help me to review if there is a better solution to solve the issue? Thanks again!
David

Reply:
We are rather thin on ES knowledge here and up to now none of the relevant experts has weighed in yet, but, speaking for myself, the main question is how much this impacts the behaviour outside of the specific case you've been looking at (which it clearly improves). In my preliminary tests the change in behaviour is large so it will take some time to get to a definite conclusion on the PR.

Reply:
Thank you so much for your explanation and help!